### ISSUE TITLE
Remove the hardcoding of the admin profile in the profile initialization stage.

### ISSUE THREAD

**Issue by jamesxu0** [oppia/oppia-android#482]
Currently an admin is hard coded into the application. This needs to be removed when there is a proper admin account creation.

**Comment by seanlip** [comment]
Closing since this will be superseded by the updated onboarding flow.

**Comment by seanlip** [comment]
@BenHenning I'm not sure what to do with this issue. Could you please confirm whether the TODOs are obsolete at this point, and should be removed -- or do you want to keep them, still?

**Comment by BenHenning** [comment]
@seanlip I think this issue could be repurposed to track the new admin onboarding flow since that actually relates to the TODOs (which are valid concerns). Might that be something we can get from product? Ideally we'd paste in a filled out feature request template into the main comment & update the title accordingly.

**Comment by seanlip** [comment]
Hm, I would rather create a new issue for the new onboarding flow since this one seems like it's from a more technical perspective and I don't understand the relation to the TODOs.

Could you explain a bit more what it means for an "admin to be hard coded into the application" and why the new flow allows us to remove that hardcoding? I think the main problem is that I don't understand what's intended by the original issue (and TODOs) at all.

**Comment by seanlip** [comment]
Per @BenHenning: the way the profile system is initialized for the first time, it hardcodes the admin profile. The TODOs are saying "get rid of the hardcoding behaviour", but we can't do that without the new onboarding flow.

So, changing the title of this issue to "Remove the hardcoding of the admin profile in the profile initialization stage" and it'll be blocked on the new onboarding flow being implemented (see #4938).

**Comment by Cyclotron17** [comment]
@adhiamboperes are you still working on this issue???

**Comment by adhiamboperes** [comment]
> @adhiamboperes are you still working on this issue???

Yes. Additionally, this issue is marked as blocked. Please select a different issue.

**Comment by adhiamboperes** [comment]
@omkodes, please leave a comment here so that I can assign you.

**Comment by omkodes** [comment]
Hi @adhiamboperes, I'm ready to be assigned to this! I've already opened PR #6362 with the fix.
